### ISSUE TITLE
Improve interface of the "environment" input variable

### ISSUE THREAD

**Issue by CarlosDomingues** [cloudposse/terraform-aws-ecs-container-definition#61]
That's not an issue, just a suggestion.

The current way of passing environment variables to the module is by using a list of maps (`list(object({ name = string, value = string }))`) , e.g:

```hcl
module "my_container_definition" {
  source  = "cloudposse/ecs-container-definition/aws"
  version = "v0.23.0"
  [...]

  environment= [
    {
      name  = "FIRST_ENV_VAR_NAME",
      value = "FIRST_ENV_VAR_VALUE"
    },
    {
      name  = "SECOND_ENV_VAR_NAME",
      value = "SECOND_ENV_VAR_VALUE"
    },
    {
      name  = "THIRD_ENV_VAR_NAME",
      value = "THIRD_ENV_VAR_VALUE"
    } 
  ]
}
```

IMO that adds a lot of noise to code (14 lines to declare 3 environment variables), making it much less pleasant to read.

Currently I'm using the following pattern:

```hcl
module "my_container_definition" {
  source  = "cloudposse/ecs-container-definition/aws"
  version = "v0.23.0"
  [...]

  environment = [for name, value in var.environment: {"name" = name, "value" = value}]
```

Then I can declare environment variables like so:

```hcl
module "my_outer_module" {

  [...]
  environment = {
    FIRST_ENV_VAR_NAME = "FIRST_ENV_VAR_VALUE"
    SECOND_ENV_VAR_NAME = "SECOND_ENV_VAR_VALUE"
    THIRD_ENV_VAR_NAME = "THIRD_ENV_VAR_VALUE"
  }
}
```

Which is much more readable, specially for a high number of variables. 

Would anyone oppose a PR to add this pattern to the module?

**Comment by BernalCarlos** [comment]
I agree with you, I would like to see something this.

**Comment by Gowiem** [comment]
Closed by #63. Release is [0.36.0](https://github.com/cloudposse/terraform-aws-ecs-container-definition/releases/tag/0.36.0). 

cc @CarlosDomingues @BernalCarlos
